test/benchmark.js

```diff
@@ -20,7 +20,12 @@ const isWin32 = require('os').platform() === 'win32';
 const npmScript = isWin32 ? 'npm.cmd' : 'npm';
 
 const testDir = resolve('.tmp-hexo-theme-unit-test');
+const flamegraphDir = process.env.TRAVIS_BUILD_DIR
+  ? join(process.env.TRAVIS_BUILD_DIR, '0x')
+  : join(testDir, '0x');
+
 const hexoBin = resolve(testDir, 'node_modules/.bin/hexo');
```

Comment:
Are you willing to use $GITHUB_WORKSPACE?

Reply:
As I said, it is only a PoC. Simply pick the changes (not the entire commit) will be fine then.

For GitHub Action, `const flamegraphDir = join(testDir, '0x');` will be fine then.